File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsDeletePlantsRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsDeletePlantsRequestItem(
    @JsonProperty("ActualDate")
    val actualDate: String? = null,
    @JsonProperty("Count")
    val count: Int? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("Label")
    val label: String? = null,
    @JsonProperty("ReasonNote")
    val reasonNote: String? = null,
    @JsonProperty("WasteMaterialMixed")
    val wasteMaterialMixed: String? = null,
    @JsonProperty("WasteMethodName")
    val wasteMethodName: String? = null,
    @JsonProperty("WasteReasonName")
    val wasteReasonName: String? = null,
    @JsonProperty("WasteUnitOfMeasureName")
    val wasteUnitOfMeasureName: String? = null,
    @JsonProperty("WasteWeight")
    val wasteWeight: Double? = null
)

File: sdks/thunkmetrc-rs/wrapper/src/models/plants_delete_plants_request_item.rs
use serde::{Deserialize, Serialize};
#[allow(unused_imports)]
use super::*;

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct PlantsDeletePlantsRequestItem {
    #[serde(rename = "actual_date", skip_serializing_if = "Option::is_none")]
    pub actual_date: Option<String>,
    #[serde(rename = "count", skip_serializing_if = "Option::is_none")]
    pub count: Option<i64>,
    #[serde(rename = "id", skip_serializing_if = "Option::is_none")]
    pub id: Option<i64>,
    #[serde(rename = "label", skip_serializing_if = "Option::is_none")]
    pub label: Option<String>,
    #[serde(rename = "reason_note", skip_serializing_if = "Option::is_none")]
    pub reason_note: Option<String>,
    #[serde(rename = "waste_material_mixed", skip_serializing_if = "Option::is_none")]
    pub waste_material_mixed: Option<String>,
    #[serde(rename = "waste_method_name", skip_serializing_if = "Option::is_none")]
    pub waste_method_name: Option<String>,
    #[serde(rename = "waste_reason_name", skip_serializing_if = "Option::is_none")]
    pub waste_reason_name: Option<String>,
    #[serde(rename = "waste_unit_of_measure_name", skip_serializing_if = "Option::is_none")]
    pub waste_unit_of_measure_name: Option<String>,
    #[serde(rename = "waste_weight", skip_serializing_if = "Option::is_none")]
    pub waste_weight: Option<f64>,
}
